images/hubble-proto/install-protoc.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -o xtrace
+set -o errexit
+set -o pipefail
```

Comment:
note: Linter complains that this is not a POSIX compatible option. I think we want to use `#!/bin/bash` to satisfy the lint.

Reply:
These scripts are run inside alpine images. They don't have bash

Reply:
then let's either ignore the file during linting or apk add bash, whichever is easier is fine i think.

Reply:
Switched to bash.